pkg/policies/controlplane/circuitfactory/graph.go

```diff
@@ -26,6 +26,16 @@ func (circuit *Circuit) ToGraphView() *policymonitoringv1.CircuitView {
 	}
 }
 
+// TODO:
```

Comment:
@openai,

Can you help implement this todo?

Reply:
Need to make a few changes to the following code:
 1. Links should not be computed recursively by toGraphViewAux. Links within the children should not be part of links at the parent.
 2. When computing links at a tree certain links may reach up to the components in parent trees as well.
 3. Signals are uniquely identified by `Signal.SignalID()` which returns a `runtime.SignalID`:
```
type SignalID struct {
	SubCircuitID string
	SignalName   string
}
```

Reply:
Can you please share the changes here? You do not have ability to make commits yet

Reply:
Please generate go code based on above comment.